frontend/src/ramp.pug

```diff
@@ -26,9 +26,14 @@
       v-bind:title="`${tframe === 'day' ? '[' + slice.date + '] Daily' : '[' + slice.date + ' till ' + slice.endDate + '] Weekly'} contribution: ${slice.insertions} lines`",
       v-bind:href="getLink(user, slice)", target="_blank",
       v-bind:class="'ramp__slice--color' + getSliceColor(slice.date)",
-      v-bind:style="{\
-          zIndex: user.commits.length - j,\
-          borderLeftWidth: getWidth(slice) + 'em',\
-          right: (getSlicePos(tframe === 'day' ? slice.date : slice.endDate) * 100) + '%'\
-      }"
+      v-bind:style="[\
+          {\
+            zIndex: user.commits.length - j,\
+            borderLeftWidth: getWidth(slice) + 'em',\
+            right: (getSlicePos(tframe === 'day' ? slice.date : slice.endDate) * 100) + '%' \
+          },\
+          mergegroup && groupby === 'groupByAuthors' ? \
```

Comment:
Can combine with this, looks similar? The dynamic style is harder to maintain.
![image](https://user-images.githubusercontent.com/35621726/60105045-319ffa80-9795-11e9-9b2d-67e32d723651.png)

Reply:
ok noted.